### ISSUE TITLE
[Accessibility: ACC-264.1] Busy Indicator: Loading not conveyed.

### ISSUE THREAD

**Issue by divyamalhotra488** [SAP/fundamental-ngx#12600]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
Screen Loading should be conveyed to users when busy indicator is applied.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Component used: [Busy Indicator](https://sap.github.io/fundamental-ngx/#/core/busy-indicator)
When the user activates ‘Refresh’ button, the wait is indicated with 3 dots for visual users, but the same loading state is not conveyed to screen reader users.
<img width="452" alt="image" src="https://github.com/user-attachments/assets/baa558a7-6a71-47c0-8e44-25926aedd8e5">

#### Please provide stackblitz example(s).
You can try ngx examples, issue is reproducible there.

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.
Yes, issue is raised by an accessibility expert.

#### Did you check the documentation and the API?
Yes

#### Did you search for similar issues?
Yes

#### Is there anything else we should know?

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**

**Comment by droshev** [comment]
@divyamalhotra488 The screenshot and the scenario is an application and it should be handled by them. Can you address this to them?
